code-challenges.js

```diff
@@ -18,6 +62,25 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+//write a function that takes in an array
+const cubeAndSum = (array) =>  {
+    //start with a constant at 0 to add to
+    var sum = 0
+    //I found this syntax online, but I'm not totally sure how it works because I havent seen it before- it looks like a way to write a for loop but a bit different?
```

Comment:
yep! it's basically a way to write a loop that goes through the length of an array without the additional syntax.